### PR TITLE
July and August 2017 Spotlights

### PR DESCRIPTION
**Pull request by Shadowthegod**
### Description
Adds the images and article to July and August Spotlights

### Status
Finished; Pending Review 

### Changelog
- added article
- added images

### CODE REVIEW

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+
+**[Mirash](https://osu.ppy.sh/u/2841009)** has done an excellent job in keeping all elements in mind when mapping this song - the background rhythm, the vocals and the instruments. With careful placement and neat accentuation Mirash has succeeded in offering the community a map which can be enjoyed by many people.
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/zauberkugel.jpg)](https://osu.ppy.sh/s/554892)
+
+Yet again. **[pishifat](https://osu.ppy.sh/u/3178418)** manages to impress the community with a map in this month's spotlights. Zaberkugel heavily focuses on slider speed changes as the song progresses. Combining with creative slider arrangements and great gameplay overall make this one a must-play!
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/sayonara-heaven.jpg)](https://osu.ppy.sh/s/516995)
+
+With such clean and consistent mapping, **(jieusieu)[https://osu.ppy.sh/u/759439]**, **[Beomsan](https://osu.ppy.sh/u/3626063)** and **[Enon](https://osu.ppy.sh/u/2043401)** managed to create a wonderful mapset worthy of this month's Spotlight.
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: formatting is the other way around here

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+
+Each difficulty captures the song wonderfully while also putting attention to the breaks in the music, and the map itself flows brilliantly, topped off with some excellent and beatifully fitting hitsounding.
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: hitsounding →  hit sounding

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+
+Not only that, this mapset also contains a fantastic storyboard by someone who decided to remain anonymous. Definitely worth giving this mapset a try which maintains a gradual difficulty spread for players of all types.
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: of all types → of all levels?

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/heroine-syndrome.jpg)](https://osu.ppy.sh/s/603694)
+
+While it may not be innovative, **[Delis](https://osu.ppy.sh/u/1603923)** has presented us with yet another well made map this month filled, with his nostalgia-focused design choices. What makes Delis's maps stand out from the masses of modern maps is their old-style object composition and slider design, which resemble maps from an age most current osu! players have never gotten a chance to see.
+
+## osu!taiko
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/soumatou-labyrinth.jpg)](https://osu.ppy.sh/s/625493)
+
+The winner of the osu!taiko spotlights this month is no one else but **[komasy](https://osu.ppy.sh/u/1980256)**! Soumatou Labyrinth provides an extremely solid and consistent spread from Kantan to Inner Oni which will satisfy players of any skill level. Special shoutout to the Inner Oni that makes use of very intuitive 1/6 snappings while keeping an overall high difficulty with the usage of complex rhythms, suiting the song perfectly. A well deserved first place!
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: probably Spotlights needs capitalization on the S

1/6 snappings → 1/6 patterns or patterning

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+
+Comprising of 6 difficulties ranging from Futsuu to Destructive Nightmare, this mapset is an outstanding example of how high bpm maps should be handled. In spite of the song's chaotic nature, _yu68 expertly interweaves a congruent flow into his own map, making it extremely fun and satisfying to play. This map certainly isn't one for the faint-hearted though, and will definitely give all players a real challenge!
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/click-soleily-remix.jpg)](https://osu.ppy.sh/s/609391)
+
+A cool map from **[iloveyou4ever](https://osu.ppy.sh/u/4964596)** this month! This approved difficulty sufficiently utilizes characteristic sound effects of Soleily present in the song as the framework for the map, making pattern arrangement clean and consistent. Streams of moderate length in the transition periods accompany the music pretty well, which add spice to the overall structure of the map.
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: sound effects of Soleily → sound effects from Soleily ?

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 6]
```diff
@@ -0,0 +1,259 @@
...
+
+Slider velocity alterations are also natural and intuitive, fitting the pace of the song accordingly. You're sure to find it interesting and get some simple stream practice through playing this beatmap.
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/cry-thunder.jpg)](https://osu.ppy.sh/s/500752)
+
+It's time to Cry Thunder! **[Surono](https://osu.ppy.sh/u/3611370)** is storming this month's spotlights with a 1/3 and 1/6-snap filled song by DragonForce!
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: spotlights capitalisation

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 7]
```diff
@@ -0,0 +1,259 @@
...
+
+Using the relatively uncommon 1/3 snap as the song's main rhythm, Surono has created a map that becomes noteworthy simply from clever utilization of this unusual beat snapping. Mixed together with solid drum mapping and guitar solos, he has made a very respectable entry into this month's Spotlight.
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/my-so-called-life.jpg)](https://osu.ppy.sh/s/627934)
+
+Finally, three years after the first Venetian Snares song has been ranked in osu!standard, osu!taiko has finally gotten a ranked map of this breakcore artist as well, and it has proven to work very nicely.
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: has been ranked → had been ranked

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 8]
```diff
@@ -0,0 +1,259 @@
...
+
+Difficult rhythms, dense patterns, streams and various combined note snaps: My So-Called Life has everything, and **[Raiden](https://osu.ppy.sh/u/2239480)** has combined them all into an amazing map. Also, don't get fooled by the long calm section in the first third of the song - the difficulty quickly ramps up as the song enters its more intense parts.
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/sayonara-memories.jpg)](https://osu.ppy.sh/s/606364)
+
+**[Akemi_Homura](https://osu.ppy.sh/u/707980)** has put out a great entry into Ranked this month with Sayonara Memories! The simple 1/2 + 1/4 patterns are neatly mapped with appropriate consistency and the song parts were divided very well by patterning within the map itself, making it very interesting to play. It is also good practice for double time, since the map has many rest sections dividing the more difficult ones. There's bound to be something for everyone in this one!
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: capitalize Double Time to make it stand out?

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 9]
```diff
@@ -0,0 +1,259 @@
...
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/I.jpg)](https://osu.ppy.sh/s/426638)
+
+Here we have **[Tidek](https://osu.ppy.sh/u/743282)** this time with a beatmap worthy enough for the Spotlight. Knowing that **[_underjoy](https://osu.ppy.sh/u/2235750)** has made 2 guest difficulties, we can expect a big and wide-reaching work from these two experienced Polish mappers.
+
+The best thing about this map is that you can actually appreciate how the patterns and scroll speed changes gets harder at the same time in each difficulty, if you play them from easiest to hardest. This beatmap is highly recommended if you are fan of scroll speed changes and challenging patterns. Enjoy!
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/altersist.jpg)](https://osu.ppy.sh/s/459950)
+
+**[pwhk](https://osu.ppy.sh/u/4887865)** is back once again with a 7-key BMS style map. With the growing trend of fancy scroll speed changes and unique maps, sometimes it is also nice to look at maps that are kept simplistic yet deliver satisfaction.
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: BMS style → BMS-style

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 10]
```diff
@@ -0,0 +1,259 @@
...
+
+The beginning simple slowjam that develops with polyrhythms to the first kiai section is very satisfying to nail. The ending section may be a bit biased to favor the left hand space bar users, but this is a very solid map overall. Any pro 7K players who love some challenge should try this map with DoubleTime; the general patterning of this map is very Double Time-friendly yet offers a whole new level of difficulty.
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: the beginning simple → the beginning's simple

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 11]
```diff
@@ -0,0 +1,259 @@
...
+
+## osu!taiko
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/true-blue.jpg)](https://osu.ppy.sh/s/643803)
+
+Wait, what year it is? Eurobeat in 2017?! This time we have a remix of the song True Blue mapped by [iloveyou4ever](https://osu.ppy.sh/u/4964596), he previously mapped the original song and got in the spotlights too, but let's talk about this beautiful map (and song). We will see a good follow of the vocals and instrumental as well some piano, even if the map is 164 bpm you will enjoy it with a lot of streams and some high slider velocity.
+
+The song and the map itself it feels relaxing to play even with the amount of notes it has, as it contains a good structure and a consistent mapping, in my opinion, it's the kind of map that even if you manage to get full combo, you are gonna still replaying it to get a good score on it, and also, don't forget to give it a chance with HardRock and Hidden!
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: I think this line needs slight rewording. 
"The song and the map themselves feel relaxing to play even with the high amount of notes it has, as it contains a well structured and consistent mapping in my opinion. It's the kind of map that you'd still replay even after managing to get a full combo on it! Also, don't forget to give it a chance with mods like HardRock or Hidden!"

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 12]
```diff
@@ -0,0 +1,259 @@
...
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/reminiscing.jpg)](https://osu.ppy.sh/s/643269)
+
+Featured as a guest difficulty for [toybot](https://osu.ppy.sh/u/2848604)'s set, we have Reminiscing by [tasuke912](https://osu.ppy.sh/u/2774767)! An approval sized map using complex yet intuitive 1/6 and 1/3 snapping combination throughout the map, making it quite challenging for a 5.94 Star Rated beatmap.
+
+Rhythms used are fairly straightforward as we can expect from a drum and bass track, but the relatively high BPM will keep your hands hot and busy! This beatmap is definitely a must have for every high tier taiko player.
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: taiko player → osu!taiko player (sake of consistency)

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 13]
```diff
@@ -0,0 +1,259 @@
...
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/false-cross.jpg)](https://osu.ppy.sh/s/615067)
+Despite a fairly repetitive song, [butter0414](https://osu.ppy.sh/u/7935022) managed to create a set with a good variety of patterning to dodge the song’s natural tedium, with the patterning increasingly gradually to encompass quintuplets mixed with ⅙ streams.
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: patterning is used too much in a single sentence. Switch the second for density instead?

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 14]
```diff
@@ -0,0 +1,259 @@
...
+
+One of this month's three spotlighted full spread sets is [maziari1105](https://osu.ppy.sh/u/4469725)'s jazzy Din Don Dan, which won the 2nd Japanese osu!taiko beatmapping contest.
+
+This map features a variety of patterns with many of them being snapped to the uncommon off-beat rhythms which this song, being a jazz remix of a Ryu* song, has to offer. Even though it's one of the easier maps in this month's spotlight, this set is enjoyable for players and mappers alike. Generally a nice addition to the sometimes monotonous song choices you have in the ranked section of osu!taiko. Truly a good map.
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/marine-blue-ni-sotte.jpg)](https://osu.ppy.sh/s/630650)
+
+A well structured mapset by [Nardoxyribonucleic](https://osu.ppy.sh/u/876419), almost completely bereft of any issues to point out. Simply superb all around.
+
+He managed to achieve a perfect balance in his following of the vocals by ample usage of cleverly designed rest, creating an excellent portrayal of a variety of techniques that aren’t often used when mapping vocals so closely in osu!taiko. An example for mappers to take note of!
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: rest → rest moments

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 15]
```diff
@@ -0,0 +1,259 @@
...
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/black-or-white.jpg)](https://osu.ppy.sh/s/606833)
+
+Is it better to be BLACK or WHITE? The answer isn't obvious, but [Arrival](https://osu.ppy.sh/u/1694000)'s "White" difficulty stands spotlighted in a mapset that is predominantly "Black". How poetic.
+
+Despite clocking in at only 180 BPM, Arrival's map exudes an unparalleled intensity. His pattern choice is fitting, and is further complemented by his ability to weave complex rhythms throughout multiple streams. The map is a gripping 5 minute journey that takes you through the best of what taiko has to offer. A truly noteworthy addition to the monthly spotlights.
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: the best of what taiko has to offer. → the best of what osu!taiko has to offer.

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 16]
```diff
@@ -0,0 +1,259 @@
...
+
+With nearly twenty minutes of drain time on offer (more than any previously ranked osu!catch beatmap set!), this offering is a perfect fit for stamina-lovers of all skill levels. Serving up a full difficulty spread from four different mappers, each putting their own unique touch on this vibrant and beautiful song, this is truly a worthy winner of this month's Spotlight.
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/boogie.jpg)](https://osu.ppy.sh/s/569985)
+
+It has been a quiet couple of years for fans of legendary mapper [Spectator](https://osu.ppy.sh/u/702598) but now he's back and as brilliant as ever. Boogie is an intense and unrelenting beatmap that barely gives the player a moment’s rest. Fast and dense quarter beat patterns dominate the beatmap with quick hyperdashes and constant direction changes that require real stamina and concentration to catch.
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: fans of legendary → fans of the legendary

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 17]
```diff
@@ -0,0 +1,259 @@
...
+
+Despite the rather repetitive nature of the song Spectator uses just the right amount of variety to keep things familiar yet fresh that overall makes for a fantastic six minutes of frenzied fruit catching.
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: missing comma after "song"

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 18]
```diff
@@ -0,0 +1,259 @@
...
+
+## osu!mania
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/swampgator.jpg)](https://osu.ppy.sh/s/613792)
+
+Swamp, predators, alligators and... anime grills? Once again we have [Wonki](https://osu.ppy.sh/u/7898495) featured as the top of the month with, yet again, another track by Zekk. A really catchy song combined with suited scroll velocity changes makes this set overall great.
+
+It feels refreshing everytime you play it, and besides how harsh the map looks, it plays in an exceptionally smooth manner which makes it quite enjoyable and adds a good touch of replayability. Technical patterns, heavy layering and a great guest difficulty by Fontes. Everything we look for in the Spotlights!
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: besides → despite of / in spite of

File: news/2017-09-21-beatmap-spotlights-july-august-2017.md [hunk 19]
```diff
@@ -0,0 +1,259 @@
...
+
+[![](/wiki/shared/news/2017-09-21-beatmap-spotlights-july-august-2017/flower-spring.jpg)](https://osu.ppy.sh/s/300685)
+
+[Kawawa](https://osu.ppy.sh/u/4647754) is back again to bless the ranked section of mania with another 7-key marathon length map, this time with a wide variety of patterns and techniques.
```
Comment (Raidencio) [inline news/2017-09-21-beatmap-spotlights-july-august-2017.md]: mania → osu!mania

**Comment by Shadowthegod** [comment]
Fixed those, and a few more I noticed along the way.

**Pull request merged**